projects.html

```diff
@@ -5,25 +5,25 @@
 <h1 class="openlex-header">OpenLexington Projects</h1>
 <div class="development" id="current">
   <h2 class="openlex-header">In Development:</h2>
-  <a href="https://github.com/openlexington/bus-alert" class="project-title">Bus Alert</a>
+  <a href="https://github.com/openlexington/bus-alert" class="project-title" target="_blank">Bus Alert</a>
   <span class="description">Lexington mobile app to alert you when your bus is approaching  </span>
-  <a href="https://github.com/openlexington/LIVES_viewer" class="project-title">LIVES Viewer</a>
+  <a href="https://github.com/openlexington/LIVES_viewer" class="project-title" target="_blank">LIVES Viewer</a>
   <span class="description">A simple web app for viewing LIVES standard health inspection data  </span>
-  <a href="https://github.com/openlexington/councilmatic" class="project-title">Councilmatic</a>
+  <a href="https://github.com/openlexington/councilmatic" class="project-title" target="_blank">Councilmatic</a>
   <span class="description">A City Council legislative information subscription service  </span>
-  <a href="https://github.com/openlexington/Lex_to_LIVES" class="project-title">Lex to LIVES</a>
+  <a href="https://github.com/openlexington/Lex_to_LIVES" class="project-title" target="_blank">Lex to LIVES</a>
   <span class="description">Lexington Health Dept. Health Inspection data to LIVES format  </span>
-  <a href="https://github.com/openlexington/WhatsMyDistrict" class="project-title">What's My District</a>
+  <a href="https://github.com/openlexington/WhatsMyDistrict" class="project-title" target="_blank">What's My District</a>
   <span class="description">According to your home address, an app that tells you your district and other zoning information  </span>
 </div>
 
 <div class="launched" id="past">
   <h2 class="openlex-header">Launched:</h2>
-  <a href="http://whatsmydistrict.org" class="project-title">What's My District</a>
+  <a href="http://whatsmydistrict.org" class="project-title" target="_blank">What's My District</a>
   <span class="description">According to your home address, an app that tells you your district and other zoning information  </span>
-  <a href="http://lexington-wiki.org" class="project-title">Local Wiki</a>
+  <a href="http://lexington-wiki.org" class="project-title" target="_blank">Local Wiki</a>
   <span class="description">A crowdsource of local knowledge about Lexington, KY  </span>
-  <a href="http://lookatlex.org" class="project-title">Look At Lexington</a>
+  <a href="http://lookatlex.org" class="project-title" target="_blank">Look At Lexington</a>
```

Comment:
@chaserx No problem! I'm excited to get involved (on the code side at least)

The updates should be good to go.
As for design, I agree. I will start by opening issue for small layout changes to the projects page.

Cheers